src/orders/components/OrderMetadataDialog/test-ids.ts

```diff
@@ -0,0 +1,2 @@
+export const TEST_ID_ORDER_LINE_METADATA = "order-line-metadata";
+export const TEST_ID_PRODUCT_VARIANT_METADATA = "product-variant-metadata";
```

Comment:
question: Do we need to store data-test-id? Is it good practice? I'm asking because first time see something like this?

Reply:
I think storing them separately from app's code is a good idea:
- you can import these test-id's in tests, if you need to update them in app, then they don't break your tests
- you can re-use them for different set of tests if you want

I could have exported them from the same file (rather than creating `test-ids.ts`) but that would break hot-module reloading.